src/cc_catalog_airflow/dags/provider_api_scripts/metropolitan_museum_of_art.py

```diff
@@ -0,0 +1,279 @@
+    response = _get_response_json(None, endpoint)
+
+    if response:
+        total_object_ids = response['total']
+        object_ids = response['objectIDs']
+    else:
+        logger.warning(f'No content available')
+        return None
+    return [total_object_ids, object_ids]
+
+
+def _get_response_json(
+        query_params,
+        endpoint,
+        request_headers=DEFAULT_REQUEST_HEADERS,
+        retries=5, 
+):
+    response_json = None
+
+    if retries < 0:
+        logger.error('No retries remaining.  Failure.')
+        raise Exception('Retries exceeded')
+
+    response = delayed_requester.get(
+        endpoint,
+        params=query_params,
+        headers=request_headers,
+        timeout=60
+    )
+    if response is not None and response.status_code == 200:
+        try:
+            response_json = response.json()
+        except Exception as e:
+            logger.warning(f'Could not get response_json.\n{e}')
+            response_json = None
+
+    if (
+            response_json is None
+            or response_json.get('error') is not None
+    ):
+        logger.warning(f'Bad response_json:  {response_json}')
+        logger.warning(
+            'Retrying:\n_get_response_json(\n'
+            f'    {endpoint},\n'
+            f'    {query_params},\n'
+            f'    {request_headers}'
+            f'    retries={retries - 1}'
+            ')'
+        )
+        response_json = _get_response_json(
+            query_params,
+            endpoint=endpoint,
+            request_headers=request_headers,
+            retries=retries - 1
+        )
+        
+    return response_json
+
+
+def _extract_the_data(object_ids):
+    for i in object_ids:
+        _get_data_for_each_image(i)
+
+
+def _get_data_for_each_image(object_id):
+
+    endpoint = 'https://collectionapi.metmuseum.org/public/collection/v1/objects/{}'.format(object_id)
+
+    object_json = _get_response_json(None, endpoint)
+
+    if object_json is None:
+        logger.error('Unable to process object ID : {}'.format(object_id))
+        return None
+
+    message = object_json.get('message')
+    if message:
+        logger.warning(f'{message} : {object_id}')
+        return None
+
+    isCC0 = object_json.get('isPublicDomain')
+    if (isCC0 is None) or isCC0 is False:
+        logger.warning('CC0 license not detected')
+        return None
+
+    foreign_url = object_json.get('objectURL', None)
+    if foreign_url is None:
+        logger.warning(f'No landing page detected for: {object_id}')
+        return None
+
+    image_info = object_json.get('primaryImage', None)
+    if image_info is None:
+        logger.warning(f'No image found for {object_id}')
+        return None
+
+    title = object_json.get('title')
+    creator_name = object_json.get('artistDisplayName')
+    foreign_id = object_id
+    meta_data = _create_meta_data(object_json)
+    image_url = image_info
+
+    thumbnail = ''
+    if '/origina/' in image_url:
```

Comment:
I think this is a typo:  `/origina/` should be `/original/`.

If possible, please also add a test that would catch this error in the future.